testing-library/pptr-testing-library
# feat: update @library-testing/dom to 7

- Update `@testing-library/dom` to v7
- Update `wait-for-expect` to v3
- Install matching `@types/puppeteer` version on CI
- Update pptr version matrix
  - 1.5 -> 1.6 as it [doesn't seem to work ](https://travis-ci.org/github/testing-library/pptr-testing-library/builds/665007899)on node v13
  - 1.7 -> 1.8 as there is no `@types/puppeteer` v1.7
- Updated `dom-testing-library` references in readme
- Attempted to replace `waitForException` with `@testing-library/dom`'s `wait` but it requires dom access
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43102 is taking a while to merge so I removed `waitFor` untill the types are updated.
